### PR TITLE
[Application] Merge process files on both parallel and non-parallel process

### PR DESCRIPTION
**Pull request by samsonasik**
@TomasVotruba @staabm continue of PRs:

- https://github.com/rectorphp/rector-src/pull/4521
- https://github.com/rectorphp/rector-src/pull/4522

this PR merge process between parallel and non-parallel to avoid miss logic applied in the future.

### CODE REVIEW

File: packages/Parallel/WorkerRunner.php [hunk 1]
```diff
@@ -51,12 +38,12 @@ public function run(Encoder $encoder, Decoder $decoder, Configuration $configura
...
         // 1. handle system error
         $handleErrorCallback = static function (Throwable $throwable) use ($encoder): void {
-            $systemErrors = new SystemError($throwable->getMessage(), $throwable->getFile(), $throwable->getLine());
+            $systemError = new SystemError($throwable->getMessage(), $throwable->getFile(), $throwable->getLine());
 
             $encoder->write([
                 ReactCommand::ACTION => Action::RESULT,
                 self::RESULT => [
-                    Bridge::SYSTEM_ERRORS => [$systemErrors],
+                    Bridge::SYSTEM_ERRORS => [$systemError],
```
Comment (TomasVotruba) [inline packages/Parallel/WorkerRunner.php:45]: :+1:

**Comment by samsonasik** [comment]
@TomasVotruba Ready for review 👍 . `check-active-class` CI notice seems related with PR:

- https://github.com/rectorphp/rector-src/pull/4534

**Comment by samsonasik** [comment]
Rebased 👍

**Comment by samsonasik** [comment]
All checks have passed 🎉 @TomasVotruba it is ready for review.

**Comment by TomasVotruba** [comment]
Let's give this a try :) thanks :+1:

**Pull request merged**